lib/constants.ts

```diff
@@ -28,6 +28,8 @@ export const BUILD_DIR = "build";
 export const OUTPUTS_DIR = "outputs";
 export const APK_DIR = "apk";
 export const RESOURCES_DIR = "res";
+export const CONFIG_NS_FILE_NAME = "nsconfig.json";
+export const CONFIG_NS_APP_RESOURCES_ENTRY = "app_resources";
```

Comment:
I prefer naming json properties in camelCase (appResources in this case), but I do not have a strong opinion here. @KristianDD , @Mitko-Kerezov  what do you think?

Reply:
I'll follow the convention seemingly established in package.json and tsconfig and go with camelCase.